content/applications/hr/appraisals/reporting.rst

```diff
@@ -0,0 +1,188 @@
+=========
+Reporting
+=========
+
+In Odoo's *Appraisals* app, two metrics are tracked as appraisals are completed: an :ref:`appraisal
+analysis <appraisals/analysis>`, and a :ref:`skills evolution <appraisals/skills-report>`.
+
+.. _appraisals/analysis:
```

Comment:
Nitpick– please update this custom anchor to include "report" so it matches `appraisals/skills-report`
```suggestion
.. _appraisals/analysis-report:
```